html/js/display-product.js

```diff
@@ -35,22 +35,22 @@ class RobotoffAsker extends HTMLElement {
         .hidden { display: none; }
       </style>
       <div class="row hidden zoomrow" id="zoomrow">
-        <div class="medium-12 columns">
+        <div class="small-12 columns">
           <img src="" id="zoom" data-zoom-in-src="" data-zoom-out-src="">
         </div>
       </div>
       <div class="row">
-        <div class="medium-2 columns">
+        <div class="small-2 columns hidden">
           <img src="" id="thumbnail">
         </div>
-        <div class="medium-10 large-6 columns">
+        <div class="small-12 columns">
```

Comment:
```suggestion
        <div class="small-10 columns">
```
Re-enable the thumbnail. 10/12 columns seems to be enough for the texts.

Reply:
I prefer to put the text and buttons at the very left. There is the same 100% width on a position:fixed element that makes the actual available space much less than 10/12.

e.g. it can look like this with the small-2 + small-10 columns:

![image](https://user-images.githubusercontent.com/8158668/58912780-266c2880-871b-11e9-817f-4a98a8de6aba.png)